airflow/providers/google/cloud/hooks/bigtable.py

```diff
@@ -54,7 +54,7 @@ def _get_client(self, project_id: str):
         return self._client
 
     @GoogleBaseHook.fallback_to_default_project_id
-    def get_instance(self, instance_id: str, project_id: Optional[str] = None) -> Instance:
+    def get_instance(self, instance_id: str, project_id: str) -> Instance:
```

Comment:
Let's decide what we want to use: `str` or `Optional[str] = None` and be cosistent

Reply:
Yeah. Agree. project_id: str is better IMHO. it is semantically correct - we expect project_id to be set. And  it is used most of the time below.

Reply:
@jmcarp I checked the changes and in some places where we use the decorator we are using `project_id: str` and in others `project_id: Optional[str] = None`. I think we should be consistent :)